Library/Homebrew/rubocops/cask/shared_filelist_glob.rb

```diff
@@ -0,0 +1,37 @@
+# typed: true
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Cask
+      class SharedFilelistGlob < Base
+        extend AutoCorrector
+
+        def on_send(node)
+          return if node.method_name != :zap
+
+          node.each_descendant(:pair).each do |pair|
+            symbols = pair.children.select(&:sym_type?).map(&:value)
+            next unless symbols.include?(:trash)
+
+            pair.each_descendant(:array).each do |array|
+              regex = /sfl\d"$/
```

Comment:
```suggestion
              regex = /\.sfl\d"$/
```

Would it make sense to make this regex slightly more restrictive?